src/index.html

```diff
@@ -2,26 +2,49 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Pokemon</title>
 </head>
 
 <body>
-    <div class="container">
-        <header>
-            <div class="logo">
-                <img src="assets/images/logo.png">
-            </div>
-            <nav class="nav">
-                <a href="#">Planets</a>
-                <a href="#">Sapceships</a>
-                <a href="#">Races</a>
-                <a href="#">Airplanes</a>
-            </nav>
-        </header>
-    </div>
+  <div class="conatiner">
+    <header>
+      <section>
+        <a href="index.html" id="myLogo"></a>
+        <div class="navigation">
+          <a id="home" class="page-active">Pokemons <span id="pokemonCount"></span> </a>
+          <a id="about" class="">About</a>
+        </div>
+        <div class="right"></div>
+      </section>
+
+    </header>
+    <main class="container">
+      <div id="poke_container" class="poke_container row "> </div>
+      <div id="aboutContainer" class="about-container">
```

Comment:
- [ ] Please kindly keep the naming conventions consistent. I notice you are using camelCases, dash-cases, and snake_cases. Please kindly use consistent naming conventions for better readability of your code. Also kindly apply one of the naming conventions to your displaypokemonitems.js.
- [ ] For the DRY principle, please use only one poke_container either for id or class. Even though you have it in the id field, I notice you called it using querySelector in your displaypokemonitems.js. Kindly fix those ids in other files as well.